bokeh/core/_templates/autoload_nb_js.js

```diff
@@ -160,8 +160,11 @@
       if (el != null) {
         el.textContent = "BokehJS " + root.Bokeh.version + " successfully loaded.";
       }
-    } else if (Date.now() < root._bokeh_timeout) {
+    }
+    else if (Date.now() < root._bokeh_timeout) {
```

Comment:
the `else if` should be moved back up one line right after the `}` to match our repo style

Reply:
I will fix it.